jatszohaz/jatszohaz/models.py

```diff
@@ -110,3 +111,11 @@ def reset_all_checked_profile():
             user.room = ""
             user.checked_profile = False
             user.save()
+
+
+class Comment(TimeStampedModel):

```

Comment:
Lehetne `UserComment`-nek vagy hasonlónak hívni, hogy le lehessen összekeverni a kölcsönzés kommenttel.

Reply:
Illetve légyszi hozzáadod a `jatszohaz/admin.py`-ban ezt a modell it? 
`admin.site.register(Comment)`

Ez azért lenne hasznos, hogy az admin oldalon (amihez csak fejlesztők férnek hozzá) megjelenjen és meg lehessen nézni/módosítani/törölni ha valamiért igény van rá.